docs/_layouts/test-page.njk

```diff
@@ -0,0 +1,38 @@
+---
+layout: base.njk
+---
+
+<!-- this style is only for the test page -->
+<style scoped>
+  .toast         {
+    position: fixed;
+    right: 10px;
+    z-index: 3000;
```

Comment:
If the toasts are meant to be underneath the modal, I believe the range of the z-index would fall into the layer before modal being less than 1040.

Reply:
i am not worried about that on this test page